### PR TITLE
Placeholder ember-cli notes 12 october 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-cli/2017-10/october-10.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ember-cli Meeting Notes - 2017-10-10
+
+## Attendees
```
Comment (hjdivad) [inline ember-cli/2017-10/october-10.md]: Please reply here with agenda items

Comment (hjdivad) [reply ember-cli/2017-10/october-10.md]: SIGUSR1 or equiv for triggering rebuild cc @rwjblue

Comment (hjdivad) [reply ember-cli/2017-10/october-10.md]: follow-up strategies rfc

Comment (kellyselden) [reply ember-cli/2017-10/october-10.md]: release post-mortem

Comment (kellyselden) [reply ember-cli/2017-10/october-10.md]: ember-cli-update in core status

**Comment by hjdivad** [comment]
cc @kategengler @kellyselden @kratiahuja @rwjblue @trabus

File: ember-cli/2017-10/october-12.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+rj: so the problem there is that `npm install` will download optional dependencies but add them in `package.json` under `dependencies`.  All of the things have now been fixed:
+  - ember-data dev dependencies are now properly listed as dev dependencies, not dependencies
+  - hash-for-dep hashes correctly and does not error if it can't resolve a dep
+
+### ember-cli-update in core status
+
+ks: should we be putting this in release notes for ember yet?
+
+rj: i think the next step is to get our hosue in order for this beta cycle and then do that.  For instance, the ember-cli release notes don't yet say to use `ember-cli-update`.
```
Comment (alexander-alvarez) [inline ember-cli/2017-10/october-12.md]: house

Comment (hjdivad) [reply ember-cli/2017-10/october-12.md]: okay @alexander-alvarez we'll go with your more conventional spelling ^_^

**Pull request merged**